parsers/dataField/parser_test.go

```diff
@@ -317,3 +318,20 @@ func TestOperationDataFieldParser_ParseRelayed(t *testing.T) {
 		}, res)
 	})
 }
+
+func TestParseSCDeploy(t *testing.T) {
+	arguments := createMockArgumentsOperationParser()
+	parser, _ := NewOperationDataFieldParser(arguments)
+
+	t.Run("ScDeploy", func(t *testing.T) {
+		t.Parallel()
+
+		dataField := []byte("0101020304050607")
+		rcvAddr := make([]byte, 0)
```

Comment:
Can we change `createMockArgumentsOperationParser` to create a pub key converter mock with address len = 32 and use it to test it here as well?

Reply:
done.